Source/Prism.Ninject.Wpf.Tests/NinjectRegionNavigationContentLoaderFixture.cs

```diff
@@ -0,0 +1,72 @@
+using System.Linq;
+using Microsoft.Practices.ServiceLocation;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Ninject;
+using Prism.IocContainer.Wpf.Tests.Support.Mocks.Views;
+using Prism.Ninject;
+using Prism.Ninject.Wpf.Tests.Mocks;
+using Prism.Regions;
+
+namespace Prism.Unity.Wpf.Tests
+{
+    [TestClass]
+    public class NinjectRegionNavigationContentLoaderFixture
+    {
+        private MockServiceLocator mockServiceLocator;
+        private IKernel _kernel;
+
+        [TestMethod]
+        public void ShouldFindCandidateViewInRegion()
+        {
+            IKernel kernel = new StandardKernel();
+            kernel.Bind<object, MockView>().To<MockView>().Named("MockView");
+
+            ConfigureMockServiceLocator(kernel);
+
+            // We cannot access the UnityRegionNavigationContentLoader directly so we need to call its
```

Comment:
need to update this comment so it reflects that this is a test for ninject and not unity.